z390test/src/test/groovy/org/z390/test/RunRtTests.groovy

```diff
@@ -0,0 +1,101 @@
+package org.z390.test
+
+import org.junit.jupiter.api.Test
+
+class RunRtTests extends z390Test {
+    var sysmac = basePath("mac")
+    var syscpy = basePath("mac")
+    var libs = ["SYSMAC(${sysmac}+)", "SYSCPY(${syscpy}+)"]
+    // rt1 = asmlg
+    var rt1Options = ['bal', 'notiming', 'stats', *libs]
+    // rt3 = mz390
+    var rt3Options = ['noasm', 'bal', 'notiming', 'stats', *libs]
+    // rt4 = mz390
+    var rt4Options = ['noasm',  'bal', 'notiming', 'stats', *libs]
+    // rt7 = asml + ez390
+    var rt7AsmlOptions = ['bal', 'notiming', *libs]
+    var rt7Ez390Options = ['notiming', 'stats', *libs]
+
+    @Test
+    void test_TESTERR1() {
+        // rt3 = mz390
+        var syscpyRtTest = "SYSCPY(${basePath('rt', 'test')})"
+        int rc = this.mz390(basePath("rt", "test", "TESTERR1"), *rt3Options, syscpyRtTest)
```

Comment:
Why did you add the syscpyRtTest value?

Reply:
Not sure, may have been a cut and paste issue.. have removed and test still runs.